kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js

```diff
@@ -91,6 +91,41 @@ onEvent('recipes', (event) => {
             temperature: 800,
             time: 58,
             id: 'tconstruct:smeltery/melting/metal/iron/chain'
+        },
+        {
+            ingredient: { item: 'dustrial_decor:gold_chain' },
+            result: { fluid: 'tconstruct:molten_gold', amount: 22 },
+            temperature: 800,
+            time: 58,
+            id: '${id_prefix}molten_gold/chain'
+        },
+        {
+            ingredient: { item: 'betterendforge:thalassium_chain' },
+            result: { fluid: 'tconstruct:molten_thalassium', amount: 22 },

```

Comment:
This has made me realize that chains need a bit of work in general. the 22mb used above in iron chains is from the cheapened iron chain recipe. 

8 chains = 11 nuggets and 1 nugget is 16mb

So
8 chains = 176mb or 1 chain = 22mb

Not all of the chain recipes have the same values though. Would you be comfortable altering shaped crafting recipes as well to match iron/gold chains? (one craft gets 8 chains)

Reply:
I am not sure what this refers to? I thought I had made sure that other chains were the same cost, but I'll take another look.

Reply:
I have fixed the values for Thalassium, Terminite and Nebu as they were inconsistent.